src/Tribe/Attendee_Registration/Rewrite.php

```diff
@@ -18,11 +18,11 @@ public function __construct( WP_Rewrite $wp_rewrite = null ) {
 	}
 
 	/**
-	 * Generate the Rewrite Rules
+	 * Generate the Rewrite Rules.
 	 *
 	 * @since 4.9
 	 *
-	 * @param  WP_Rewrite $wp_rewrite WordPress Rewrite that will be modified, pass it by reference (&$wp_rewrite)
+	 * @param  WP_Rewrite $wp_rewrite WordPress rewrite that will be modified, pass it by reference (&$wp_rewrite).
```

Comment:
Does not need to be passed by reference

```suggestion
	 * @param  WP_Rewrite $wp_rewrite WordPress rewrite that will be modified.
```